ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -76,6 +76,132 @@
         en: 'Knockback',
       },
     },
+    {
+      id: 'Copied Hobbes Laser-Resistance Test',
+      regex: / 14:4805:Hobbes starts using Laser-Resistance Test/,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer' || data.CanAddle();
+      },
+      infoText: {
+        en: 'aoe',
+      },
+    },
+    {
+      id: 'Copied Hobbes Right Arm',
+      regex: / 00:0839:The wall-mounted right arm begins to move\.\.\./,
+      run: function(data) {
+        data.alliance = data.alliance || 'A';
+      },
+      infoText: {
+        en: 'Dodge Moving Circle',
+      },
+    },
+    {
+      id: 'Copied Hobbes Flamethrowers',
+      regex: / 00:0839:The wall-mounted flamethrowers activate\./,
+      run: function(data) {
+        data.alliance = data.alliance || 'B';
+      },
+      alertText: {
+        en: 'Dodge Flamethrowers',
+      },
+    },
+    {
+      id: 'Copied Hobbes Left Arm 1',
+      regex: / 00:0839:The wall-mounted left arm begins to move\.\.\./,
+      durationSeconds: 6,
+      run: function(data) {
+        data.alliance = data.alliance || 'C';
+      },
+      infoText: {
+        en: 'Out',
+      },
+    },
+    {
+      id: 'Copied Hobbes Left Arm 2',
+      regex: / 00:0839:The wall-mounted left arm begins to move\.\.\./,
+      delaySeconds: 8,
+      alertText: {
+        en: 'Dodge Falling Walls',
+      },
+    },
+    {
+      id: 'Copied Hobbes Left Arm 3',
+      regex: / 00:0839:The wall-mounted left arm begins to move\.\.\./,
+      delaySeconds: 10,
+      alertText: {
+        en: 'Spread Tethers',
+      },
+    },
+    {
+      id: 'Copied Hobbes Short Missile',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:00C4:/,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      alertText: {
+        en: 'Spread',
+      },
+    },
+    {
+      id: 'Copied Hobbes Ring Laser',
```

Comment:
I know this is maybe contradictory, but I think things that are just ground aoes don't need triggers, and this could be removed.